### PR TITLE
Ssb: Add 5gen

### PR DESCRIPTION
**Pull request by Volco**
We really need to fix the commit history somehow lol

### CODE REVIEW

**Comment by Volco** [comment]
No serious this commit history needs fixing... it's bad lmao

File: mods/seasonal/random-teams.js [hunk 1]
```diff
@@ -41,6 +41,12 @@ class RandomStaffBrosTeams extends RandomTeams {
 			// Nature needs to be a valid nature with the first letter capitalized ex: Modest
 			*/
 			// Please keep sets organized alphabetically based on staff member name!
+			'5gen': {
```
Comment (ghost) [inline mods/seasonal/random-teams.js]: 2 < 5 so it should go after 2xTheTap imo

Comment (Volco) [reply mods/seasonal/random-teams.js]: `>> ["2", "5", "5", "2"].sort()
<< ["2", "2", "5", "5"]`
no

Comment (Volco) [reply mods/seasonal/random-teams.js]: wait

Comment (Volco) [reply mods/seasonal/random-teams.js]: okay true lmao fixing

Comment (Volco) [reply mods/seasonal/random-teams.js]: ~~when you fail to read orderings properly~~

**Comment by HoeenCoder** [comment]
Like I said, I had to do a rebase pull to fix the build for @ImLycan the best way to fix this is to save your changes in another file, nuke the branch locally, and recreate

You could also create a new clean ssb3 branch and cherry pick commits from this onto that

**Pull request closed without merging**